### PR TITLE
MAT-7722: Resizable Library Editor

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-7722](https://jira.cms.gov/browse/MAT-7722)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editCqlLibrary/EditCQLLibrary.scss [hunk 1]
```diff
@@ -10,56 +10,10 @@
...
 
-  #header-editor-row {
-    display: flex;
-    flex-direction: row;
-    flex-grow: 1;
-    height: 48px;
-    justify-content: flex-end;
-    background: #ededed;
-  }
-
```
Comment (mcmcphillips) [inline src/components/editCqlLibrary/EditCQLLibrary.scss]: This looks important elsewhere

**Review by mcmcphillips — CHANGES_REQUESTED** [review]
image

Check your changes against the other cql editors to confirm there hasn't been any breaking changes.

These changes appear to have significantly affected cql-editor

File: src/components/editCqlLibrary/EditCqlLibrary.tsx [hunk 2]
```diff
@@ -356,203 +358,199 @@ const EditCqlLibrary = () => {
...
+        <Allotment>
+          <Allotment.Pane>
+            <div id="left-panel" tw="flex-grow">
+              <IconButton
+                data-testid="editor-search-button"
+                aria-label="search button"
                 style={{
-                  display: "flex",
-                  justifyContent: "center",
-                  height: "calc(100vh - 135px)",
```
Comment (mcmcphillips) [inline src/components/editCqlLibrary/EditCqlLibrary.tsx:392]: Lets keep the old calc until we implement a programmatic check for making sure it's always the right size

**Review by mcmcphillips — APPROVED** [review]
